<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsoleOutput</class>
 <widget class="QWidget" name="ConsoleOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelConsole">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Console</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="textEditConsole">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string>[SIMULATION] Exécution des commandes...
[IN] tdmsimport + computekey
</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonStop">
       <property name="text">
        <string>⏹ Arrêter</string>
       </property>
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Arrêter l'exécution en cours</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEffacer">
       <property name="text">
        <string>Effacer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExporter">
       <property name="text">
        <string>Exporter</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
